backend/prompt_studio/prompt_studio_core_v2/serializers.py

```diff
@@ -69,7 +73,26 @@ def to_representation(self, instance):  # type: ignore
             if prompt_instance.count() != 0:
                 for prompt in prompt_instance:
                     prompt_serializer = ToolStudioPromptSerializer(prompt)
-                    output.append(prompt_serializer.data)
+                    prompt_outputs = (
```

Comment:
@jagadeeswaran-zipstack move this coverage calculation logic as a method within the `PromptStudioOutputManager` class. That way, we can avoid code duplication and also let such logic of aggregation / querying be in the model layer itself and serializer can purely be a matter of validation and data representation

Reply:
Done :+1: